packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/template.tsx

```diff
@@ -9,18 +9,20 @@ import {
   SceneHeaderText,
   SceneSubHeaderText
 } from 'components/Layout'
+import { ModalName } from 'data/modals/types'
 
+import { Props } from '.'
 import { IntroCardWrapper, Wrapper } from './model'
 
-const OrderMyCardBtn = () => (
+const OrderMyCardBtn = ({ onClick }) => (
```

Comment:
This onClick needs to be typed, otherwise, for every time I see this component, I'll have to think:

1. Does this function accept a parameter?
2. Is it an async function? Should I wait for this?
3. Is it required? Can I just remove it?

Reply:
And we don't call `buttons` as `bts`, please use the full name.

Reply:
fixed